gradio/cli/commands/components/dev.py

```diff
@@ -8,7 +8,7 @@
 
 import gradio
 
-gradio_template_path = Path(gradio.__file__).parent / "templates" / "dev"
```

Comment:
I thought we had to modify the `pyproject.toml` for the package to work properly? Something about the `tool.hatch.build` section.

Sorry wrong place to put this diff lol

Reply:
Yeah we do, I'll change that

Reply:
done.

Reply:
Nice!